molecule/command/init/role.py

```diff
@@ -38,6 +38,17 @@ class Role(base.Base):
     .. option:: molecule init role --role-name foo
 
         Initialize a new role.
+
+    .. program:: molecule init role --role-name foo --template path
+
+    .. option:: molecule init role --role-name foo --template path
+
+        Initialize a new role using a local *cookiecutter* template. This
+        allows the customization of a role while still using the default
+        generated ``molecule`` folder. This is similar to an
+        ``ansible-galaxy init`` skeleton. Please refer to the ``init scenario``
```

Comment:
Might be better to rephrase to:

Please refer to the ``init scenario`` command in order to generate a custom ``molecule`` scenario.

Since you aren't customizing the default scenario since it already exists, right?

Reply:
I  would also change 

> default generated ``molecule`` folder

  to

> "upstream ``molecule`` folder"

like you said on your other change request. It does feel more appropriate. Thanks!

Reply:
Done